### ISSUE TITLE
Request to Fix the Root Cause of the Overflow Menu Accessibility Errors

### ISSUE THREAD

**Issue by emilyz-github** [carbon-design-system/carbon#8431]
## Environment
> Operating system
Mac OS Catalina 10.15.6

> Browser
Google Chrome

> Automated testing tool and ruleset
IBM Equal Access Accessibility Checker

## Detailed description
We request to reopen #8100 and #8078. Please provide a complete solution.  The `main` tag workaround doesn't work for us because:
- The workaround introduces new and more violations.  
- It also caused additional unexpected issues by wrapping the content it's not supposed to wrap, for example, screen reader testing.
- Adding main tag or roles causes unnecessary styling change in our code
- Even if it solves the AVT scan, the workaround doesn't use the `main` tag correctly to serve its purpose and intention.

We use the overflow menus a lot in our code. We can't keep using the workaround. Please fix the root cause.

> What version of the Carbon Design System are you using?
"carbon-components-react": "7.30.0",

> What did you expect to happen?
Expect a real fix to the overflow menu itself.

> What happened instead?
Reported exactly the same violations as shown in #8100 and #8078

> What WCAG 2.1 checkpoint does the issue violate?
"2.4.1 Bypass blocks"

## Steps to reproduce the issue
Same as reported in #8100 and #8078

**Comment by tw15egan** [comment]
Hey there @emilyz-github , since this issue deals with where the OverflowMenu is placed in your application, we can't provide a solution for you out of the box. The `OverflowMenu` needs to reside _somewhere_ in a landmark region on your page. If your HTML structure looks like `<body><main> ... your app code </main></body>`, you should only need the one `<main>` element as long as each `OverflowMenu` is rendered inside of that landmark.

If that is not possible, you can take a look at the other landmark elements and see which one makes sense for your overflow menu to reside in. Here is a link to the WCAG spec, as well as some example pages

Example page that displays landmarks (click show landmarks at the top): https://www.w3.org/TR/wai-aria-practices/examples/landmarks/main.html

General information: https://www.w3.org/TR/wai-aria-practices/examples/landmarks/index.html

Hope that helps!

**Comment by tw15egan** [comment]
Closing as resolved

Ref https://github.com/carbon-design-system/carbon/issues/7305
